sdk/ai/azure-ai-inference/src/main/java/com/azure/ai/inference/implementation/models/EmbedRequest1.java

```diff
@@ -0,0 +1,277 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+package com.azure.ai.inference.implementation.models;
+
+import com.azure.ai.inference.models.EmbeddingEncodingFormat;
+import com.azure.ai.inference.models.EmbeddingInput;
+import com.azure.ai.inference.models.EmbeddingInputType;
+import com.azure.core.annotation.Fluent;
+import com.azure.core.annotation.Generated;
+import com.azure.json.JsonReader;
+import com.azure.json.JsonSerializable;
+import com.azure.json.JsonToken;
+import com.azure.json.JsonWriter;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * The EmbedRequest1 model.
+ */
+@Fluent
+public final class EmbedRequest1 implements JsonSerializable<EmbedRequest1> {
```

Comment:
Looks like EmbedRequest1 and EmbedRequest model above are identical. Why do we have duplicate models?

Reply:
I will remove the EmbedRequest1 and add customization to remove it on autogen

Reply:
Update, I will rename the class to ImageEmbedRequest